xbmc/addons/AddonStatusHandler.cpp

```diff
@@ -38,8 +38,13 @@ namespace ADDON
 
 CCriticalSection CAddonStatusHandler::m_critSection;
 
-CAddonStatusHandler::CAddonStatusHandler(const std::string &addonID, ADDON_STATUS status, std::string message, bool sameThread)
-  : CThread(("AddonStatus " + addonID).c_str()),
+CAddonStatusHandler::CAddonStatusHandler(const std::string& addonID,
+                                         uint32_t instanceID,
+                                         ADDON_STATUS status,
+                                         std::string message,
```

Comment:
`const std::string& message`

Reply:
Think will create a separate request to Kodi about it, as already before as this.